bittide-instances/bin/Shake.hs

```diff
@@ -19,6 +19,8 @@ import System.Console.ANSI (setSGR)
 import System.Directory (getCurrentDirectory)
 import System.FilePath (isDrive, (</>), takeDirectory)
 
+import Paths_bittide_instances
```

Comment:
Why is this separated from the rest?

Reply:
Hiddes own judgement? 😁

[Style guide](https://clash-lang.readthedocs.io/en/latest/hacking-on-clash/style.html#imports):
> Put a blank line between each group of imports. Create subgroups per your own judgement.

Although the style guide would say `Clash.Shake.Extra` and `Development.Shake.Extra` need to be moved from _standard library imports_ to _local application/library specific imports_.